src/resources/Autofill.ts

```diff
@@ -0,0 +1,47 @@
+import { AxiosInstance } from "axios";
+import { Method, Resource } from "../Resource";
+
+export type ExtractionResult = {
+  documentId: string;
+  documentClassification: {
+    issuingCountry: string;
+    documentType: string;
+    issuingState?: string;
+  };
+  extractedData: {
+    documentNumber?: string;
+    firstName?: string;
+    lastName?: string;
+    middleName?: string;
+    fullName?: string;
+    gender?: string;
+    dateOfBirth?: string;
+    dateOfExpiry?: string;
+    nationality?: string;
+    mrzLine1?: string;
+    mrzLine2?: string;
+    mrzLine3?: string;
+    addressLine1?: string;
+    addressLine2?: string;
+    addressLine3?: string;
+    addressLine4?: string;
+    addressLine5?: string;
+  };
+};
+
+export type AutofillResource = {
```

Comment:
Not sure there's any need to export this since you just pass the documentId directly as an argument to perform